src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
-
 import Bookings from "./Bookings";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
+  let address = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      {/* <header className="App-header">CYF Hotel</header> */}
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer address={address} />
```

Comment:
good use of components here, you're importing them & passing the address as props to use in your footer component, well done :)